dev/TabView/TabView.xaml

```diff
@@ -75,74 +90,23 @@
-                                                                Style="{StaticResource RepeatButtonRevealStyle}"
-                                                                Visibility="Collapsed"
-                                                                Content="&#xE76B;"/>
-
-                                                            <ScrollContentPresenter x:Name="ScrollContentPresenter"
-                                                                Grid.Column="1"
-                                                                Padding="{TemplateBinding Padding}"
-                                                                contract4Present:TabFocusNavigation="Once" />
-
-                                                            <RepeatButton x:Name="ScrollIncreaseButton"
-                                                                Grid.Column="2"
-                                                                VerticalAlignment="Bottom"
-                                                                HorizontalAlignment="Center"
-                                                                Width="{ThemeResource TabViewItemScrollButtonWidth}"
-                                                                Height="{ThemeResource TabViewItemScrollButtonHeight}"
-                                                                Padding="{ThemeResource TabViewItemScrollButtonPadding}"
-                                                                Background="Transparent"
-                                                                BorderThickness="1"
-                                                                Delay="50"
-                                                                FontFamily="Segoe MDL2 Assets"
-                                                                Interval="100"
-                                                                Style="{StaticResource RepeatButtonRevealStyle}"
-                                                                Visibility="Collapsed"
-                                                                Content="&#xE76C;"/>
-                                                        </Grid>
-                                                    </Border>
-                                                </ControlTemplate>
-                                            </Setter.Value>
-                                        </Setter>
-                                    </Style>
-                                </ScrollViewer.Style>
+                                Style="{StaticResource TabScrollViewerStyle}">
 
                                 <ItemsPresenter x:Name="TabsItemsPresenter"
                                     Padding="{TemplateBinding Padding}" />
 
                             </ScrollViewer>
 
-                            <ContentPresenter
+                            <Button
```

Comment:
>Button [](start = 29, length = 6)

I'd recommend the add button have a tooltip with something like "New tab" or "Add new tab" in it - Chrome and Edge do this for their + button for example.

Reply:
OK, makes sense.